src/constants.py

```diff
@@ -23,16 +23,17 @@
 
 METRICS_PORT = 9127
 PGB_LOG_DIR = "/var/log/pgbouncer"
-MONITORING_PASSWORD_KEY = "monitoring_password"
 AUTH_FILE_DATABAG_KEY = "auth_file"
 CFG_FILE_DATABAG_KEY = "cfg_file"
 
 EXTENSIONS_BLOCKING_MESSAGE = "bad relation request - remote app requested extensions, which are unsupported. Please remove this relation."
 CONTAINER_UNAVAILABLE_MESSAGE = "PgBouncer container currently unavailable"
 
-SECRET_LABEL = "secret"
-SECRET_INTERNAL_LABEL = "internal-secret"
-SECRET_DELETED_LABEL = "None"
+# Labels are not confidential
+SECRET_LABEL = "secret"  # noqa: S105
+MONITORING_PASSWORD_KEY = "monitoring_password"  # noqa: S105
+SECRET_INTERNAL_LABEL = "internal-secret"  # noqa: S105
+SECRET_DELETED_LABEL = "None"  # noqa: S105
```

Comment:
Security checks will inevitably produce some false positives, so we will have to `noqa` them. I propose adding a comment on top of `noqa` blocks, so that we explain why the check is disabled.